src/body.rs

```diff
@@ -109,6 +109,36 @@ impl Body {
         }
     }
 
+    /// Create a `Body` from a Vec of bytes.
+    ///
+    /// The Mime type is set to `application/octet-stream` if no other mime type has been set or can
+    /// be sniffed. If a `Body` has no length, HTTP implementations will often switch over to
+    /// framed messages such as [Chunked
+    /// Encoding](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Transfer-Encoding).
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use http_types::{Body, Response, StatusCode};
+    /// use async_std::io::Cursor;
+    ///
+    /// let mut req = Response::new(StatusCode::Ok);
+    ///
+    /// let input = vec![1, 2, 3];
+    /// let len = input.len();
+    /// req.set_body(Body::from_bytes(input, Some(len)));
+    /// ```
+    pub fn from_bytes(
+        bytes: Vec<u8>,
```

Comment:
So.. I guess ideally this would be `&[u8]`? However, that conflicts with the `'static` requirement of `reader: Box<dyn BufRead + Unpin + Send + Sync + 'static>`. I'm not really _why_ `Vec` doesn't conflict with that requirement either?

Reply:
This should probably take a `Vec<u8>` since it needs to keep ownership.